consensus/src/aggregator.rs

```diff
@@ -109,14 +110,18 @@ impl<V: StepVote> Aggregator<V> {
             return Err(AggregatorError::DuplicatedVote);
         }
 
-        // Check if the provisioner voted for a different result
-        let voters_list = self.uniqueness.entry(msg_step).or_default();
-        match voters_list.get(signer.bytes()) {
-            None => voters_list.insert(*signer.bytes(), v.clone()),
-            Some(prev_vote) => {
-                return Err(AggregatorError::ConflictingVote(prev_vote.clone()))
-            }
-        };
+        if v.header().iteration < EMERGENCY_MODE_ITERATION_THRESHOLD {
```

Comment:
Please add a comment about why we disable collecting faults and allowing double voting when in emergency mode